### ISSUE TITLE
Priority 1 accessibility issue: Non-text content did not have a text alternative that served the equivalent purpose.

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6541]
**What**

The text alternative for the comic did not describe all of the information conveyed within the image. The text within the image was not described exactly and the images were not described. The meaning and flow of the content was lost as the distinction between the information within each of the four panels was not distinguishable from the alt text ‘Web comic comprising four panels showing how the human body is a temple, a shrine to every feeling about your appearance, alters of confusing health advice and a place to sit down and eat a sandwich in peace.’.

Current Code Ref(s): #image-gallery-1 > div.container > div > div > div > div.Space-sc- 1ntp1c3-0.fSTAlJ > figure > div > img
<img width="3000" height="3000" class="lazy-image bg-charcoal font-white image ls- is-cached lazyloaded" [...] alt="Web comic comprising four panels showing how the human body is a temple, a shrine to every feeling about your appearance, alters of confusing health advice and a place to sit down and eat a sandwich in peace. " [...]>

**Solution:**
Please ensure all non-text content provides a suitable alternative that describes its purpose. 

A long description which describes the visual content and the text within each of the four panels could be offered adjacent to the image and signposted from the image. 

For more information, please visit G74: Providing a long description in text near the non-text content, with a reference to the location of the long description in the short description.

**Where**

URL: https://wellcomecollection.org/articles/YBMARRMAACEAk26C 
Page title: Temple | Wellcome Collection

**Comment by pollecuttn** [comment]
https://wellcome.slack.com/archives/C3N7J05TK/p1625656260067200

"the final issue was that the comics don't have suitable alt-text that conveys fully the content and the tone of the image. Ben has experience in writing alt-text and we are also going to look into others to consult on best-practice, so that we can do this better in future. We don't currently have capacity in the team to go back through and fix the archive of all the comics. I think it was mentioned that perhaps it might be a possible task for an intern or work experience student to work on, ad hoc., at some time in the future, but no detailed plans there to deal with that."

To be done by @dannybirchall 's team when they have capacity.

**Comment by jennpb** [comment]
@dannybirchall Any recent update on this one?

**Comment by dannybirchall** [comment]
@jennpb we've got better alt-text practice for comics now, see eg:

https://wellcomecollection.org/articles/YzQT_BEAANURiuK9

**Comment by jennpb** [comment]
Nice. So we can close this as fixed, yes? 

cc @alicerichmond

**Comment by dannybirchall** [comment]
we sure can!
